dao/src/test/java/com/iluwatar/dao/CustomerTest.java

```diff
@@ -0,0 +1,87 @@
+package com.iluwatar.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class CustomerTest {
+
+	private Customer customer;
+	private static final int ID = 1;
+	private static final String FIRSTNAME = "Winston";
+	private static final String LASTNAME = "Churchill";
+	
+	@Before
+	public void setUp() {
+		customer = new Customer(ID, FIRSTNAME, LASTNAME);
+	}
+	
+	@Test
+	public void getIndex() {
```

Comment:
Yes, I agree. Testing setters and getters gives marginally better test coverage but increases code mass and maintenance effort. Overall the effect is negative.